vivisect/__init__.py

```diff
@@ -391,6 +391,12 @@ def isFunctionSignature(self, va):
         offset, bytes = self.getByteDef(va)
         return self.sigtree.isSignature(bytes, offset=offset)
 
+    def addNoReturnVa(self, va):
+        noretva = self.getMeta('NoReturnApisVa', {})
+        noretva[va] = True
+        self.cfctx.addNoReturnAddr(va)
+        self.setMeta('NoReturnApisVa', noretva)
```

Comment:
somewhat aesthetic, but partly functional, is there any reason to add to cfctx before setMeta?
this may seem idiotic, but it's a break in the flow, and while self.cfctx.addNoReturnAddr() may never throw an exception, if it does, neither thing gets updated.  
brass-tacks: it's aesthetically pleasing to swap these last two lines and make this into a couple "functional units"

Reply:
love the addition to the api, tho! :+1:

Reply:
Aye. I'll swap 'em.